Dockerfile

```diff
@@ -90,6 +90,33 @@ RUN apt-get purge -y --auto-remove -o APT::AutoRemove::RecommendsImportant=false
 
 CMD ["/bin/bash"]
 
+# `docs_italia_it_celery_web`: Build image for celery-web
+# We need additional packages to run the document converter
+
+FROM docs_italia_it_web AS docs_italia_it_celery_web
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+        unzip \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN mkdir /tmp/converter
+ADD https://github.com/italia/docs-italia-comandi-conversione/releases/download/v0.6/converti.zip /tmp/converter
```

Comment:
è fattibile usare una variabile per dichiarare la versione dei comandi di conversione e dei filtri?

Reply:
Fixed in https://github.com/italia/docs.italia.it/pull/287/commits/2a77382b627ffc083726090439e9b44e39389f90